keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/CheckScheduler.kt

```diff
@@ -94,8 +95,12 @@ class CheckScheduler(
                    */
                   withTimeout(checkTimeout.toMillis()) {
                     launch {
```

Comment:
General comment - have you thought about how we communicate this to a user? since we will "freeze" the environment while it's being verified, is there a way we can let the user know? (maybe this is just UI treatment if a verification is running)

Reply:
Thought: we could take advantage of `ApplicationEvent`s and start adding more sub-types, which we can then surface in the UI. We already have an API to query for them (`GET /application/{application}/events`).

Reply:
I like the idea of adding this via events.